packages/backend/src/open_payments/payment_pointer/model.ts

```diff
@@ -84,6 +85,20 @@ export class PaymentPointer
       }
     }
   }
+
+  public toOpenPaymentsType({
+    authServer
+  }: {
+    authServer: string
+  }): OpenPaymentsPaymentPointer {
+    return {
+      id: this.url,
```

Comment:
Would returning `url: this.url` be more clear from an Open Payments API perspective?

Reply:
As in, change `id` to `url` in the Open Payments spec?
https://github.com/interledger/open-payments/blob/b363d33038fe789e5388f04f80ddd06a4fa97093/openapi/resource-server.yaml#L809-L813

Reply:
> As in, change id to url in the Open Payments spec?

Exactly, wasn't a comment of the code, more in general. `id`s for other resources (incoming/outgoing payments) have a different meaning that the `id` for the payment pointer.

Reply:
:thinking: Want to elaborate on how the `id` is different for the payment pointer (here or in an [`open-payments`](https://github.com/interledger/open-payments) issue)?
Part of me wants to say payment pointers are probably going to be out of scope for Open Payments soon.
Also:
- https://github.com/interledger/open-payments/pull/223

Reply:
Well, for example in our case, we call incoming payments by `baseUrl/incoming-payment/{{id}}`, id being a uuid, a sub resource identifier. You don't necessarily have a similar "baseUrl/payment-pointer" resource, where you could GET a payment pointer using the `id` path param. 

Likewise in `backend`, when we use `paymentPointer.id` its usually the model's uuid. I don't think it's worth changing, just might not be always 100% clear, is all.